.github/workflows/ci.yaml

```diff
@@ -0,0 +1,32 @@
+name: Tests
+
+on: pull_request
```

Comment:
@Dreamsorcerer I think I figured it out... So it looks like since we don't run GHA on `push`, there's nothing reporting coverage for `master` anymore which in turn may affect what coverage reports.

Reply:
https://app.codecov.io/gh/aio-libs/aiohttp-session/ — no updated coverage after merging this PR